backend/tournesol/tests/test_suggestions.py

```diff
@@ -359,7 +394,7 @@ def test_most_informative_vid_given_first(self):
         )
         last_vid_score = 1000
         for v in user2_videos:
-            v_score = v.score_computation(user_videos[0]) + v.graph_sparsity
+            v_score = v.score_computation(user_videos[0]) + v.graph_sparsity(user_videos[0])
```

Comment:
Do we really want to call `.graph_sparsity()` here, now that it's included into `.score_computation()`

Reply:
Absolutely !